### DISCUSSION TITLE
Lib docs update?

### DISCUSSION THREAD

**Discussion by moon-dew** [Rapptz/discord.py#7008] [General]
Do you have a timeframe for when the official docs will be updated? I'm super excited to use the new components and slash commands (when finished), but I'm not super experienced, so I'm not really able to read the raw code well enough to write my own, so I would love the docs to be updated with that soon, or even just add an example or two of usage in there!

**Comment by Zomatree** [comment]
The docs will be updated when 2.0 is released.

**Reply by moon-dew** [reply]
Great, thank you!

**Comment by Rapptz** [comment]
The docs for the in-progress master branch is placed in a [separate page](https://discordpy.readthedocs.io/en/master/). It'll be published under `/latest/` and `/stable/` when it actually releases.
